LISAv2-Framework.psm1

```diff
@@ -81,6 +81,7 @@ function Start-LISAv2 {
 			$maxDirLength = 32
 			$workingDirectory = (Get-Location).Path
 			if ($workingDirectory.Length -gt $maxDirLength) {
+				Write-LogWarn "The path of current working directory '$workingDirectory' is too long ($workingDirectory.Length)."
```

Comment:
($workingDirectory.Length) => $($workingDirectory.Length)
Current output - The path of current working directory 'C:\jenkins-slave-executor-3\workspace\Azure\job-azure-cloudtest-manual' is too long (C:\jenkins-slave-executor-3\workspace\Azure\**job-azure-cloudtest-manual.Length**).